### ISSUE TITLE
ContainerLinux 1745.3.1 stable not successfully booting via iPXE

### ISSUE THREAD

**Issue by trevex** [coreos/bugs#2439]
# Issue Report #

In our fully PXE booted environment (no install). The stable 1745.3.1 is not booting. The previous version 1688.5.3 worked flawlessly. The ignition configuration and the image itself is managed by matchbox.
From the matchbox logs I can verify the GET requests for the ignition configs where successful.

### Container Linux Version ###

```
1745.3.1 (image version)
```

### Environment ###

ContainerLinux is used in our bare-metal university kubernetes installation. All servers are very homogeneous supermicro servers.

### Expected Behavior ###

Normal boot behaviour.

### Actual Behavior ###

The system doesn't boot and there are two units specifically that fail:
`ignition-files` and later on `initrd-cleanup`
This is all just the tty output, the machine itself is stuck afterwards and a few minutes later systemd shutsdown the system.

### Reproduction Steps ###

  1. Add several empty ContainerLinux configs with an append section:
```
ignition:
  config:
    append:
      - source: ${ignition_endpoint}?profile=kubelet&mac=${mac}
```
  2. Boot the machine

### Other Information ###

I have recorded the whole tty output via IPMI, if necessary I could upload the videos somewhere. If you need any other information let me know.

**Comment by lucab** [comment]
Thanks for the report. If you have access to the console, can you please log into the emergency shell (by hitting enter) and gather [ignition logs](https://coreos.com/ignition/docs/latest/getting-started.html#gathering-logs)?

**Comment by trevex** [comment]
Thank you, for the fast reply. Unfortunately I won't be back in the office until monday and the university  infrastructure is only accessible on premises, but a colleague is going to have a look tomorrow.

**Comment by bgilbert** [comment]
Meanwhile, if you'd like to upload the video, we'll take a look.

**Comment by trevex** [comment]
The issue was fixed by 1745.4.0. It seems to happen because of the network interface name changes, although it is unclear to me why. This is not reproducible using a VM, but rather only with the specific bare-metal and fibre bond setup.

**Comment by lucab** [comment]
Thanks for the followup! It was a duplicate of https://github.com/coreos/bugs/issues/2437 then.

As a sidenote, such kind of custom environments are unfortunately harder for us to reproduce on our CI, so if you could it would be better to throw a spare alpha/beta node in your cluster and let us know about potential regressions before they hit stable.
